tests/missing_cces.py

```diff
@@ -41,12 +58,16 @@ def check_all_rules(root):
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(
         description="Checks the all rules that are part of at least one "
-        "profile have a CCE assigned.")
+        "profile have a CCE assigned."
```

Comment:
```suggestion
        "profile have a CCE assigned. "
```

Reply:
Should be handled in [96c4dff](https://github.com/ComplianceAsCode/content/pull/9155/commits/96c4dff87ea430928d5e8e9c4f7ecccf7fbe7356)